deployments/container/Dockerfile.ubi8

```diff
@@ -51,10 +52,10 @@ RUN mkdir /licenses && mv /NGC-DL-CONTAINER-LICENSE /licenses/NGC-DL-CONTAINER-L
 COPY --from=build /artifacts/k8s-cc-manager /usr/bin/k8s-cc-manager
 COPY --from=build /artifacts/kubectl /usr/bin/kubectl
 
+ARG GPU_ADMIN_TOOLS_VERSION
 # Setup CC tools and licenses
 COPY --from=build /artifacts/cc-manager.sh /usr/bin/cc-manager.sh
-RUN curl -o /usr/bin/gpu_cc_tool.py -L https://raw.githubusercontent.com/NVIDIA/nvtrust/main/host_tools/python/gpu_cc_tool.py && chmod a+x /usr/bin/gpu_cc_tool.py
-RUN curl -o /licenses/LICENSE.txt -L https://raw.githubusercontent.com/NVIDIA/nvtrust/main/host_tools/python/LICENSE.txt
+RUN git clone https://github.com/NVIDIA/gpu-admin-tools.git /usr/bin/gpu-admin-tools && chmod a+x /usr/bin/gpu-admin-tools/nvidia_gpu_tools.py
```

Comment:
where is GPU_ADMIN_TOOLS_VERSION used here? Were you planning to checkout the specific tag?

Reply:
Does it make sense to curl just the py file from a versioned URL? Are there other files required in the repo? (also, should the `.py` file not already be executable when cloned?)

Reply:
> where is GPU_ADMIN_TOOLS_VERSION used here? Were you planning to checkout the specific tag?

@shivamerla Yes I forgot to use it in the git clone. Fixed it.

Reply:
> Does it make sense to curl just the py file from a versioned URL? Are there other files required in the repo? (also, should the `.py` file not already be executable when cloned?)

@elezar Cloning the repo becase there are other dependencies in that repo. Removed the `chmod`